cmd/main.go

```diff
@@ -111,6 +112,11 @@ func main() {
 	log.SetLogLevel(verbosity)
 	log.Info("start log metric exporter", "path", dir)
 
+	defer func() {
+		if r := recover(); r != nil {
```

Comment:
Do you have a concrete case of this program crashing with a `panic`? I wonder where it comes from, as this goroutine will only continue to create the HTTP server, which already has its own recovery mechanism.